src/community/jms-cluster/jms-geoserver/src/main/java/org/geoserver/cluster/impl/utils/JMSPropertyPlaceholderConfigurer.java

```diff
@@ -40,24 +41,23 @@ public JMSPropertyPlaceholderConfigurer(Resource defaultFile,
 
 	@Override
 	public void afterPropertiesSet() throws Exception {
-		File properties = new File(config.getConfiguration(EmbeddedBrokerConfiguration.EMBEDDED_BROKER_PROPERTIES_KEY).toString());
-		if (!properties.isAbsolute() && !properties.isFile()) {
-			// try to resolve as absolute
-			properties = new File(JMSConfiguration.getConfigPathDir(),properties.getPath());
+		org.geoserver.platform.resource.Resource properties = 
+		        Resources.fromPath(config.getConfiguration(EmbeddedBrokerConfiguration.EMBEDDED_BROKER_PROPERTIES_KEY).toString(),
+		                JMSConfiguration.getConfigPathDir());
+		if (properties.getType() != Type.RESOURCE) {
 			// copy the defaults
 			
 			InputStream inputStream = null;
 			try{
 				inputStream = defaults.getInputStream();
-				IOUtils.copy(inputStream, properties);
+				IOUtils.copy(inputStream, properties.out());
 			}finally{
 				if(inputStream!=null){
 					org.apache.commons.io.IOUtils.closeQuietly(inputStream);
 				}
 			}
 		}
```

Comment:
@NielsCharlier please feel free to fix all tabs in this method. You have made enough changes that you are welcome to fix the formatting. GitHub should still do a nice job of showing the non-whitespace changes.